<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reminder</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabBatch">
       <attribute name="title">
        <string>Запуски</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="lblDir">
            <property name="text">
             <string>Папка:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnOpenDir">
            <property name="text">
             <string>Открыть...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="editDir">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeView" name="treeBatch">
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="uniformRowHeights">
           <bool>true</bool>
          </property>
          <attribute name="headerCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Рассылка</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLineEdit" name="editFrom">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>адрес от...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editLogin">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>логин...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editPass">
           <property name="text">
            <string/>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="placeholderText">
            <string>пароль...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editHost">
           <property name="text">
            <string>mail.pulsarnpp.ru</string>
           </property>
           <property name="placeholderText">
            <string>хост...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editSMTP">
           <property name="text">
            <string>587</string>
           </property>
           <property name="placeholderText">
            <string>SMTP порт...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editIMAP">
           <property name="text">
            <string>7993</string>
           </property>
           <property name="placeholderText">
            <string>IMAP порт...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="btnSendEmails">
           <property name="text">
            <string>Выслать</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="lblTemplate">
             <property name="text">
              <string>Шаблон:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="textTemplate"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="lblAddress">
             <property name="text">
              <string>Адресаты:</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QListView" name="listAddress"/>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QPushButton" name="btnAddAddress">
                 <property name="maximumSize">
                  <size>
                   <width>24</width>
                   <height>24</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>+</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnDelAddress">
                 <property name="maximumSize">
                  <size>
                   <width>24</width>
                   <height>24</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>-</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1026</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
